CHANGELOG.md

```diff
@@ -2,8 +2,18 @@
 
 ## 0.6.4
 
-Add changelog, homepage and source code metadata so Rubygems can create links
+[Added]
+
+- This changelog file
+- Rubygems metadata in gemspec so other libs can parse this changelog
```

Comment:
Not sure how useful these entries are. I would keep the changelog as short as possible, containing only the _really_ important things.

Reply:
Well, it is important, since it solves an issue: 

The objection was (see below) that a changelog could not be found. We now not only have one, but it is accessible in a standardized way. 

> ![screenshot](https://user-images.githubusercontent.com/625294/50842901-fedd8380-1367-11e9-9bcc-8b1241b6d27c.png)